adobe/aio-lib-runtime
# [ACNA-1106][ACNA-1107] Omit old logs while -tail argument, for aio logs commands type

## Description

Prevent displaying old logs while --tail arguments is passed to `aio logs` type commands.
This will be applied to: `aio app logs --tail` and `aio runtime activation logs --tail `

While --tail argument, it should only show logs for new invocations that occur AFTER the command is run


In order to call the `printLogs` function, a condition must be met. 
The `startTime` value should be lower than the `lastActivationTime` value. 
Using the` --tail` argument, on the first run the `startTime` is `0` which makes the old logs to be displayed. 
  
`startTime` will be initialized with `Date.now()` when `--tail` is passed.

## Related Issue





https://github.com/adobe/aio-cli-plugin-runtime/issues/225
https://github.com/adobe/aio-cli-plugin-app/issues/385
https://github.com/adobe/aio-cli-plugin-app/issues/277

## Motivation and Context



## How Has This Been Tested?



Steps:
1. cd into the generated firefly project.
2. run `aio runtime activation logs --tail`.
3. While `aio runtime activation logs --tail` is running, generate some logs via the firefly app.
Expected: Only logs following the `aio runtime activation logs --tail` should be printed.

Repeat the above with: `aio app logs --tail`.
 The same behavior is expected. 

1. cd firefly project.
2. run `aio runtime activation` logs
Expected: Logs prior to `aio runtime activation` should be printed.

Repeat the above with: `aio app logs`. 
The same behavior is expected. 

## Screenshots (if appropriate):

## Types of changes




## Checklist:
